docker-compose.yml

```diff
@@ -41,23 +41,23 @@ services:
       - "./logs/:/var/www/MISP/app/tmp/logs/:delegated"
       - "./files/:/var/www/MISP/app/files/:delegated"
       - "./ssl/:/etc/nginx/certs/:delegated"
+      - "./gnupg/:/var/www/MISP/.gnupg/:delegated"
```

Comment:
`ansible-environments` should be updated as well to reflect these changes

Reply:
ACK